File: 4080-smallest-missing-multiple-of-k/smallest-missing-multiple-of-k.py
class Solution:
    def missingMultiple(self, nums, k):
        s = set()
        for num in nums:
            if num % k == 0:
                s.add(num)

        j = 1
        while True:
            value = j * k
            if value not in s:
                return value
            j += 1

File: 4080-smallest-missing-multiple-of-k/smallest-missing-multiple-of-k.cpp
class Solution {
public:
    int missingMultiple(vector<int>& nums, int k) {
        unordered_set<int> set;
        for(int i = 0; i < nums.size(); i++){
            if(nums [i] % k == 0){
                set.insert(nums [i]);
            }
        } 
        int j = 1;
        while(true){
            int value = j * k;
            if(set.find(value) == set.end()){
                return value;
            }
            j++;
        }       
    }
};